src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -56,11 +56,11 @@ setup_coverage() {
     # In CI environment we cannot install coverage in system, so we install it in a dedicated venv
     # so only coverage is available when adding this path.
     cd $TEMP_DIR
-    virtualenv --python=${TEUTHOLOGY_PYTHON_BIN:-/usr/bin/python} coverage-venv
+    virtualenv --python=/usr/bin/python3 coverage-venv
```

Comment:
unless teuthology is python3 compatible, we cannot switch over to python3 yet. see line 47 in this source file.

Reply:
@tchaikov This change does not affect teuthology virtualenv as coverage has its own virtualenv:
tested locally and it works.

Reply:
ahh, sorry! i missed the "coverage-venv" parameter.